### PR TITLE
Unify officer cards.

### PR DESCRIPTION
**Pull request by dtjanaka**
Addresses https://github.com/uclaacm/website/issues/264

### CODE REVIEW

File: pages/impact.js
Comment (advaithg) [inline pages/impact.js:81]: Maybe this should be using `&nbsp;` instead of `{' '}`?

File: pages/jedi/meet.js
Comment (advaithg) [inline pages/jedi/meet.js:40]: Same comment on `&nbsp;`

File: pages/jedi/meet.js
Comment (advaithg) [inline pages/jedi/meet.js:71]: Same comment on `&nbsp;`

**Review by advaithg — CHANGES_REQUESTED** [review]
Since the card files were removed, did you also remove the now irrelevant SCSS styling from `jedi.module.scss` and `impact.module.scss`?

**Review by mattxwang — CHANGES_REQUESTED** [review]
Exciting stuff! I'll just comment with some overall things to consider:

1. When possible, I would take advantage of the `children` prop; you can write a unified wrapper for mobile / column-wrapping behaviour (a "base" officer card of sorts), and then fill in the relevant information that differs based on the context as a child (ex "Class of...", position information, committee etc. This [composition vs inheritance](https://reactjs.org/docs/composition-vs-inheritance.html#specialization) section in the React docs explains the "React way" of doing this!
2. If you want to simplify the props chain, you can create a schema for an "Officer" object that has some mandatory fields (name, pronouns, position, ...) and then pass that in. Optional props like committee or bio can then be passed in, but you can simplify usage a little bit! 
3. The styling for the compact leadership card is not very good; feel free to rework this however you want!
4. For the tab whitespace changes, you may get a cleaner diff if you open a different PR with just the whitespace changes, rebase this PR off that branch, and then merge that first. Thanks for catching the tab inconsistency!

**Comment by dtjanaka** [comment]
> Since the card files were removed, did you also remove the now irrelevant SCSS styling from `jedi.module.scss` and `impact.module.scss`?

Done.

**Review by mattxwang — APPROVED** [review]
I'll leave it to Advaith to give a final review, but I personally think the changes look + work well! Solid work as always, Dylon! 

(feel free to punt the feedback from my previous review somewhere else - I know it's quite a bit)

**Review by advaithg — CHANGES_REQUESTED** [review]
Just noticed an error with the bigger officer cards when viewing it on smaller viewports!
The image sizes for the cards on the dev team page / the impact page remain the same even when resizing to tablet view.

impact page tablet view in deploy preview
dev team page tablet view in deploy preview

as opposed to what they should look like:

impact page tablet view on website
dev team page tablet view on website

**Review by advaithg — COMMENTED** [review]
Other than that, it looks good - so we should be good to merge once that's fixed! I'll take into account Matt's previous review as long-term things we should do or issues we can give interns / tla dev team!

**Review by advaithg — COMMENTED** [review]
Feel free to re-request review once you're done with the changes!

**Review by advaithg — APPROVED** [review]
looks good! thanks for working on this Dylon!

**Comment by advaithg** [comment]
@matthewcn56  make sure you take a look at this pr and merge main into your branch before updating leadership!

**Comment by matthewcn56** [comment]
alright will take a looksie!

**Pull request merged**
